utf8proc.c

```diff
@@ -186,6 +186,37 @@ UTF8PROC_DLLEXPORT utf8proc_ssize_t utf8proc_encode_char(utf8proc_int32_t uc, ut
 }
 
 /* internal "unsafe" version that does not check whether uc is in range */
```

Comment:
Any non-`static` functions should be prefixed with `utf8proc_` in order to prevent namespace pollution.